### PR TITLE
Feature: Added support for "Thin acrylic" system backdrop

### LINKED ISSUE #15045: Feature: Support "Thin acrylic" backdrop
### Requirements

- Add support for "Thin acrylic"

### Files Version

3.3

### Windows Version

Windows 11 23H2 22635.3350

### Comments

_No response_

### PR DESCRIPTION
**Pull request by heftymouse**
This PR adds all the available system backdrop options - None, Mica non-alt, and base Acrylic in addition to the existing Mica Alt and Thin Acrylic (called just Acrylic in the UI) options. Design aside, this is mainly for completeness in theming. Mica Alt is still the default option.

**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #15045

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files
2. Tested switching system backdrops
3. Tested switching themes

When switching themes with the system backdrop set to None, the background only updates after a resize or focusing another window. This is an issue with the way WinUI handles repainting the background.

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -3796,4 +3796,7 @@
   <data name="VerticalAlignment" xml:space="preserve">
     <value>Vertical alignment</value>
   </data>
-</root>
+  <data name="ThinAcrylic" xml:space="preserve">
+    <value>Thin Acrylic</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:3800]: Can you add a comment providing context for translators

**Review by yaira2 — APPROVED** [review]

**Review by 0x5bfa — CHANGES_REQUESTED** [review]

File: src/Files.App/ViewModels/Settings/AppearanceViewModel.cs [hunk 2]
```diff
@@ -44,12 +44,10 @@ public AppearanceViewModel(IUserSettingsService userSettingsService, IResourcesS
...
 			];
 
-			// TODO: Re-add Solid and regular Mica when theming is revamped
-			//BackdropMaterialTypes.Add(BackdropMaterialType.Solid, "Solid".GetLocalizedResource());
-
+			BackdropMaterialTypes.Add(BackdropMaterialType.Solid, "None".GetLocalizedResource());
 			BackdropMaterialTypes.Add(BackdropMaterialType.Acrylic, "Acrylic".GetLocalizedResource());
-
-			//BackdropMaterialTypes.Add(BackdropMaterialType.Mica, "Mica".GetLocalizedResource());
+			BackdropMaterialTypes.Add(BackdropMaterialType.ThinAcrylic, "ThinAcrylic".GetLocalizedResource());
```
[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/ViewModels/Settings/AppearanceViewModel.cs:49]: This sounds like something we can be doing for the other settings view models as well.

File: src/Files.App/Strings/en-US/Resources.resw [hunk 3]
```diff
@@ -3796,6 +3796,9 @@
   <data name="VerticalAlignment" xml:space="preserve">
     <value>Vertical alignment</value>
   </data>
+  <data name="ThinAcrylic" xml:space="preserve">
+    <value>Thin Acrylic</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:3800]:
```suggestion
    <value>Thin Acrylic</value>
    <comment>This is a type of backdrop for the application background</comment>
```

File: src/Files.App/ViewModels/Settings/AppearanceViewModel.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/ViewModels/Settings/AppearanceViewModel.cs:49]: Wait we can't use it for dictionary.
We can use `LocalizedEnumDescriptionFactory` we made when we implemented Layout Picker

[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/ViewModels/Settings/AppearanceViewModel.cs:49]: Please leave as is for now. I’ll make a bulk change.

**Pull request merged**